src/views/ReviewConfirm.vue

```diff
@@ -9,20 +9,20 @@
 
     <section class="mt-10">
       <header>
-        <h2>Review</h2>
+        <h2>Review and Confirm</h2>
+        <p class="mt-1">Review the information in your application. If you need change or complete anything, return to
+          the step make the necessary change.</p>
```

Comment:
Typo - missing word or something.

Reply:
Referring to the `If you need change`? => `If you need to change` ?

Reply:
oh maybe `the step make` => `the step to make` as well

Reply:
Both fixed